content/docs/gto/command-reference/index.md

```diff
@@ -0,0 +1,29 @@
+# Using GTO Commands
+
+GTO is a command line tool. Here, we provide the specifications, complete
+descriptions, and comprehensive usage examples for different `gto` commands.
+
+For a list of all commands, type `gto -h`.
+
+## Read the registry
+
+- Show the registry state, highest version, or what's assigned in stage with
+  `gto show`
+- Find out the artifact version registered/assigned with ref with
```

Comment:
Hmmm... No need to have a full list of all commands in the index I think? This is more elaborate than what's in https://mlem.ai/doc/command-reference, for example. I think it's better to stick to the **Typical GTO workflow** commands here as well, to let users know which ones are "more important" (esp. since the nav is in alphabetical order).

Reply:
Maybe something like

## Typical GTO workflow

- Make a registry with x and y commands
- Read the registry with ...
- Modify artifacts with ...
- Manage artifact enrichments with ...

Reply:
p.s. unclear if you just land here what the difference between "Modify artifacts" and "Manage artifact enrichments" is though...